index.html

```diff
@@ -4,7 +4,7 @@
     <title>Fedora API Specification</title>
     <meta charset='utf-8'>
     <meta name="description" content="Fedora API Specification">
-    <script src='https://www.w3.org/Tools/respec/respec-w3c-common'
+    <script src='http://fedora.info/spec/respec/v14.0.13/respec-w3c-common'
```

Comment:
I think the _best_ solution is to add HTTPS support to `fedora.info`, put the javascript file there and link to it using HTTPS. The downside of this is that it requires more server-side setup; the upside is that `fedora.info` _should_ use HTTPS anyway.

The _easiest_ solution is to use a relative link. The downside is, as @awoods described, there may be issues loading the javascript (I have not ever encountered this issue with Chrome or Firefox and cannot reproduce such an error using the `file://` protocol); the upside is that it requires no server-side changes.

Either way is fine with me -- it's the current text in this PR that isn't OK (it would cause the editor's draft to be considerably less usable).